mode-lambda/backend/gl/ngl.fragment.glsl

```diff
@@ -19,15 +19,19 @@ void main(void)
   vec4 SpriteColor;
 
   vec2 texSize = textureSize(SpriteAtlasTex, 0);
-  SpriteColor =
+  if (true) {
```

Comment:
I think that this `true` path is the same as what this code used to be, but I only changed it because an ES device was weird. Maybe this `true` should be `@(if (gl-es?) "false" "true")`, because this change shouldn't have anything to do with the smoothing.

Reply:
I thought calling texture() did smoothing (if enabled) while calling texelfetch() did not?

Reply:
That's right, but then this means that we can just enable/disable smoothing in that one spot and not affect the shader.

Reply:
I'm confused.  The shader has to be affected to know whether to call texture() or texelFetch().  Or maybe you are saying we always call texture() unless it is a gl-es? thing, in which case we give up on smoothing?  That makes sense to me - I will do that in the next attempt.

Reply:
Yes, that's what I mean. (Maybe this ES device I tested on was just busted anyways.)